### ISSUE TITLE
Work Item Relationships

### ISSUE THREAD

**Issue by AlanMcBee** [DarqueWarrior/vsteam#228]
Feature request (and I may try to do this?)

It seems that the only way to do anything with a work item relationship to another work item is by adding a new work item (Add-VSTeamWorkItem), and then it's only adding a relationship to an existing work item as its parent.

There would be several operations that I'd like to see here. I'm sure others would fit.

- Add-VSTeamWorkItemLink
- Get-VSTeamWorkItemLink
- Remove-VSTeamWorkItemLink

Maybe these (because I think these cases would be very common, and adding the "hierarchy-forward" or "hierarchy-reverse" link types are a little less than intuitive:
- Get-VSTeamWorkItemParent
- Get-VSTeamWorkItemChild

**Comment by DarqueWarrior** [comment]
Hey @AlanMcBee feel free to give it a try. Just follow the coding style in the other functions.

**Comment by SebastianSchuetze** [comment]
@AlanMcBee do you still want to try this?

**Comment by CodeCharm** [comment]
@SebastianSchuetze Yes. My schedule flew out of control earlier this year. I'm beginning my work on it now.

**Comment by AlanMcBee** [comment]
@SebastianSchuetze Yes. My schedule flew out of my control earlier this year. I'm starting work on it now.

**Comment by AlanMcBee** [comment]
@DarqueWarrior @SebastianSchuetze I started the Help for one of the new cmdlets (as your contribution guide suggests). I would like your take on how I'm approaching it. What works best, discussion in this issue, or a work-in-progress PR, or ...?
Here is the first pass on Add-VSTeamWorkItemLink: https://github.com/AlanMcBee/vsteam/blob/master/.docs/Add-VSTeamWorkItemLink.md

**Comment by SebastianSchuetze** [comment]
Looks good to me! 
One thing only is, that you do not need to write the syntax. It will be generated on build time. Also for the parameter description, which are pretty good. I don't know how the formatting will look like with running

```powershell
Get-Help Add-VSTeamWorkItemLink -Full
```

if it looks good there, then it is fine to me. The best way would be to look at other help files and how it is done there. If you add more details and description then it is much better anyways.

**Comment by AlanMcBee** [comment]
I am actively working on this issue, btw. Would you like to change the status from 'help wanted' ? Or do you need a PR first, even if it's WIP / incomplete?

**Comment by SebastianSchuetze** [comment]
Just commit when you think it's ready. The pipeline are running first anyways and then you know better if your code is good enough :D.

**Comment by SebastianSchuetze** [comment]
@AlanMcBee  you had a [branch ](https://github.com/AlanMcBee/vsteam/tree/WorkItemLink-228) to work on this. Could you finish it? Or maybe do you just want to open the PR in draft to see if we can help to finish it?

**Comment by CodeCharm** [comment]
@SebastianSchuetze Let me see what I can get completed by the end of the week (2020-08-08); I'll put in a PR (maybe in draft) by then. Is that acceptable?

**Comment by SebastianSchuetze** [comment]
Yes of course. Maybe I can have a look and finish it. Thank you already!

**Comment by SebastianSchuetze** [comment]
@AlanMcBee @CodeCharm  could you publish the PR soon?
